README.md

```diff
@@ -1 +1,12 @@
 # az-devops-tools
+
+## Getting Started
+
+```sh
+git clone https://github.com/wgoehrig/az-devops-tools/tree/log-tools
```

Comment:
```suggestion
git clone https://github.com/wgoehrig/az-devops-tools
```

Reply:
Whoops, sorry. Should've caught this one.